npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -138,8 +122,8 @@ export async function generateTsConfig (devServerConfig: WebpackDevServerConfig,
     includePaths.push(toPosix(cypressConfig.supportFile))
   }
 
-  if (projectConfig.architect.build.options.polyfills) {
-    const polyfills = getProjectFilePath(projectConfig.architect.build.options.polyfills)
+  if (buildOptions?.polyfills) {
+    const polyfills = getProjectFilePath(buildOptions?.polyfills)
```

Comment:
```suggestion
  if (buildOptions.polyfills) {
    const polyfills = getProjectFilePath(buildOptions.polyfills)
```

No need for the optional access here